lib/util/md5-hash.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
```

Comment:
Can you update this file to follow conventions from other files? http://eslint.org/docs/developer-guide/code-conventions.html